app/lib/permissions.js

```diff
@@ -65,7 +65,7 @@ function checkPermissionOrUserId(required_permission) {
     if(req.params.user_id == req.user.user_id) return next();
     if(hasPermission(req,required_permission)) return next();
     var p = required_permission;
-     return res.status(400).render('permissionsError.pug',{required_permission:p,required_user:req.params.user_id});
+     return res.status(400).render('user_permissionError.pug',{required_permission:p,required_user:req.params.user_id});
```

Comment:
Nitpick:  mix between camel and pascal case.  JavaScript world typically standardizes on camel case.

Reply:
Noted - will deal with that in the future ... there's quite a few other pug files with the same mix.  Personally I like that it makes it clear the broad ("user") vs. specific ("permissionError") purpose of the file, but that's just me I suppose.